genestack_client/data_importer.py

```diff
@@ -336,12 +375,14 @@ def create_sequencing_assay(self, parent, name=None, urls=None,
         :rtype: str
         """
         metainfo = metainfo or Metainfo()
-        name and metainfo.add_string(Metainfo.NAME, name)
-        organism and metainfo.add_string(BioMetaKeys.ORGANISM, organism)
-        method and metainfo.add_string(BioMetaKeys.METHOD, method)
+        self.__add_to_metainfo(metainfo, Metainfo.NAME, name, StringValue, required=True)
+        self.__add_to_metainfo(metainfo, BioMetaKeys.ORGANISM, organism, StringValue)
+        self.__add_to_metainfo(metainfo, BioMetaKeys.METHOD, method, StringValue)
         if urls:
+            urls = (url for url in urls)
+            self.__add_to_metainfo(metainfo, BioMetaKeys.READS_LINK, urls.next(), ExternalLink)
             for url in urls:
-                metainfo.add_external_link(BioMetaKeys.READS_LINK, url)
+                metainfo.add_value(BioMetaKeys.READS_LINK, ExternalLink(url))
```

Comment:
So, what's the purpose of this change (and similar ones)?  `add_external_link` calls `add_value` under the hood anyway, so I see no reason to make these changes.